pydicom/dataelem.py

```diff
@@ -527,6 +527,8 @@ def _convert(self, val: object) -> object:
             return UID(val) if val is not None else None
         elif self.VR == "PN":
             return PersonName(val)
+        elif self.VR == "AT" and val:
```

Comment:
Does this handle a tag of 0x00000000?

Reply:
Yes, you are right ... I'll amend it to check explictly for None or `''`